src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -5,19 +5,43 @@
  * максимального та мінімального елементів.
  */
 public class MatrixOperation {
-    public int[] getDiagonal(int[][] matrix) {
-        return null;
+    public int [] getDiagonal(int[][] matrix) {
+        int [] principalDiagonal = new int [matrix.length];
```

Comment:
```suggestion
        int[] principalDiagonal = new int[matrix.length];
```
please refactor your code same to this suggestion

Reply:
done